// Question 136:
for ( let i = 1; i <= 5; i++ ) {
    console.log(`Iteration: ${i}, i value, i`)
}

// Question 137:
try {
    throw new Error("Something went wrong.")
} catch (error) {
    console.log(error.message)
}